src/ExceptionHandler.php

```diff
@@ -336,26 +327,4 @@ private function getDebugMeta(Throwable $exception): array
             'trace'     => $trace,
         ];
     }
-
-    protected function getReportContext(Throwable $e): array
-    {
-        $context = [
-            'trace' => array_slice($e->getTrace(), 0, 5),
-            'file'  => $e->getFile(),
-            'line'  => $e->getLine(),
-        ];
-
-        if ($e instanceof MultiErrorInterface) {
-            $context['multi_error_errors'] = array_map(function (JsonSchemaErrorInterface $error) {
-                return [
-                    'code'   => $error->getErrorCode(),
-                    'title'  => $error->getTitle(),
-                    'detail' => $error->getDetail(),
-                    'source' => $error->getSource(),
-                ];
-            }, $e->getErrors());
-        }
-
-        return $context;
-    }
```

Comment:
This will break Rollbar reports in the API, we kind of need it.

Reply:
See https://github.com/MyParcelCOM/api/blob/develop/app/Exceptions/ExceptionHandler.php#L74

Reply:
I see, I have reverted part of the implementation!